WMF Framework/MWKDataStore+LanguageVariantMigration.swift

```diff
@@ -65,7 +65,7 @@ extension MWKDataStore {
     private func migrateSearchLanguageSetting(toLanguageVariants languageMapping: [String:String]) {
         let defaults = UserDefaults.standard
         if let url = defaults.url(forKey: WMFSearchURLKey),
-           let languageCode = url.wmf_language {
+           let languageCode = url.wmf_languageCode {
             let searchLanguageCode = languageMapping[languageCode] ?? languageCode
             defaults.wmf_setCurrentSearchContentLanguageCode(searchLanguageCode)
```

Comment:
I have noticed a few instances of method arguments named with "contentLanguageCode", yet we are passing in a wmf_languageCode and not wmf_contentLanguageCode. Could that cause any bugs?

You can do a find on this PR for "contentLanguageCode" to see other placements, I decided against commenting on all of them.

Reply:
The case I am aware of is the refactored RTL methods use `contentLanguageCode` in the name, but we pass in language code.

Fixing that is essentially the core of the Phab ticket for supporting the Arabic variant of Kazakh.

In this case, the existing search language setting will only have language codes, and if there is no migration to a variant the original code is the correct value.

When I work on the RTL issue, I can also do a double check that we aren't passing in the wrong thing elsewhere when contentLanugageCode is an argument. But, generally, since these are all naming changes, they should all be correct as-is, except for the RTL methods.